scripts/config/pages.js

```diff
@@ -118,5 +118,6 @@ module.exports = [
     // ==================== Section: "landing_pages" ====================
     // ['graduates',                               'landing_pages/graduate_program',              null,        'Binary.com Graduate Program',        'NOT-en'],
     // ['hackathon',                               'landing_pages/hackathon',                     null,        'Hackathon Competition',              'NOT-en'],
-    ['landing/signup-frame',                    'landing_pages/signup_frame',                  null,        'Sign up'],
+    ['introducing-usb',                            'landing_pages/usb',                           null,        'Introducing USB'],
```

Comment:
if we are only releasing this for english, we should add the 'NOT-en' again to the end